docs/docs/pricing/README.md

```diff
@@ -9,137 +9,150 @@ We believe anyone should be able to run simple, low-volume workflows at no cost.
 
 To support these goals, **Pipedream offers a [generous free tier](#developer-tier)**. You can run sources and workflows for free within the limits of the free tier. If you hit these limits, you can upgrade to one of our [paid tiers](#professional-tier).
 
-Read more about our plans and pricing options below.
+[Read more about our plans and pricing here](https://pipedream.com/pricing).
 
-[[toc]]
+## Free Tier
 
-## Developer Tier
+You can run up to 3 workflows and add up to 3 connected accounts for up to 100 credits a day for free, as long as they operate under [Pipedream platform limits](/limits/).
```

Comment:
The workflow and connected account limits are actually different ([see doc](https://docs.google.com/spreadsheets/d/1Hkx4M1Eo5L2WNWO9F2X6VVRJp3y6JThH4J4UMjQlT54/edit#gid=0)). I like how you've included a section on the free and paid tiers, but I would just note that Pipedream _has_ a free tier, and push people to the pricing page.